test/fixtures/CreatePeriodSummaryFixture.scala

```diff
@@ -0,0 +1,513 @@
+      |     "periodStartDate": "2019-08-24",
+      |     "periodEndDate": "2019-08-24"
+      |    },
+      |    "periodIncome": {
+      |      "turnover": 1000.99,
+      |      "other": 1000.09
+      |    }
+      |}
+    """.stripMargin
+  )
+
+  val mtdExpensesValidBodyJson: JsValue = Json.parse(
+    """
+      |{
+      |   "periodDates": {
+      |     "periodStartDate": "2019-08-24",
+      |     "periodEndDate": "2019-08-24"
+      |    },
+      |    "periodIncome": {
+      |      "turnover": 1000.99,
+      |      "other": 1000.99
+      |    },
+      |    "periodAllowableExpenses": {
+      |      "costOfGoodsAllowable": 1000.99,
+      |      "paymentsToSubcontractorsAllowable": 1000.99,
+      |      "wagesAndStaffCostsAllowable": 1000.99,
+      |      "carVanTravelExpensesAllowable": 1000.99,
+      |      "premisesRunningCostsAllowable": -99999.99,
+      |      "maintenanceCostsAllowable": -1000.99,
+      |      "adminCostsAllowable": 1000.99,
+      |      "businessEntertainmentCostsAllowable": 1000.99,
+      |      "advertisingCostsAllowable": 1000.99,
+      |      "interestOnBankOtherLoansAllowable": -1000.99,
+      |      "financeChargesAllowable": -1000.99,
+      |      "irrecoverableDebtsAllowable": -1000.99,
+      |      "professionalFeesAllowable": -99999999999.99,
+      |      "depreciationAllowable": -1000.99,
+      |      "otherExpensesAllowable": 1000.99
+      |    },
+      |    "periodDisallowableExpenses": {
+      |      "costOfGoodsDisallowable": 1000.99,
+      |      "paymentsToSubcontractorsDisallowable": 1000.99,
+      |      "wagesAndStaffCostsDisallowable": 1000.99,
+      |      "carVanTravelExpensesDisallowable": 1000.99,
+      |      "premisesRunningCostsDisallowable": -1000.99,
+      |      "maintenanceCostsDisallowable": -999.99,
+      |      "adminCostsDisallowable": 1000.99,
+      |      "businessEntertainmentCostsDisallowable": 1000.99,
+      |      "advertisingCostsDisallowable": 1000.99,
+      |      "interestOnBankOtherLoansDisallowable": -1000.99,
+      |      "financeChargesDisallowable": -9999.99,
+      |      "irrecoverableDebtsDisallowable": 1000.99,
+      |      "professionalFeesDisallowable": 9999999999.99,
+      |      "depreciationDisallowable": -99999999999.99,
+      |      "otherExpensesDisallowable": 1000.99
+      |     }
+      |}
+    """.stripMargin
+  )
+
+  val mtdAllowableConsolidatedExpensesOnlyJson: JsValue = Json.parse(
+    """
+      |{
+      |   "periodDates": {
+      |     "periodStartDate": "2019-08-24",
+      |     "periodEndDate": "2019-08-24"
+      |    },
+      |   "periodIncome": {
+      |      "turnover": 1000.99,
+      |      "other": 1000.09
+      |    },
+      |   "periodDates": {
+      |     "periodStartDate": "2019-08-24",
+      |     "periodEndDate": "2019-08-24"
+      |    },
+      |    "periodAllowableExpenses": {
+      |      "consolidatedExpenses": 1000.99,
+      |      "costOfGoodsAllowable": 1000.99,
```

Comment:
should this not be only "consolidated expenses" rather than the rest of the allowable expenses?

Reply:
so this one is consolidated expenses with only allowableExpenses and not disallowableExpenses(I admit it could be worded better)